test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DacFx/DacFxServiceTests.cs

```diff
@@ -882,6 +882,17 @@ public async Task ValidateSavePublishProfile()
             string profileFilePath = Path.Combine(folderPath, fileName);
             string expectedFile = Path.Combine(publishProfileFolder, fileName);
 
+            DeploymentOptions deploymentOptions = DeploymentOptions.GetDefaultPublishOptions();
+
+            // a few extra options get added in the publish.xml file, because of the defaults that are set in STS for maintaining compatibility with ADS and SSDT 
+            // these defaults are defined in ..\sqltoolsservice\src\Microsoft.SqlTools.ServiceLayer\DacFx\Contracts\DeploymentOptions.cs
+            deploymentOptions.ExcludeObjectTypes = new DeploymentOptionProperty<string[]>(new[] { Enum.GetName(ObjectType.Views) });
```

Comment:
`nameof(ObjectType.Views)` occurs at compile-time, so it'll be more performant overall.

More blathering: I remembered some issue with `Enum.GetName` where it would return the wrong name if there were multiple enums with overlapping values (i.e. if an enum is being used as a flag/set), so I throw a search to the ether on whether `Enum.GetName()` or `myEnum.ToString()` is better, and found [this post](https://stackoverflow.com/questions/483794/convert-enum-to-string) that told me that this third option is best!